CHANGELOG.md

```diff
@@ -24,16 +27,25 @@ instructions below to upgrade your Postgres.
 * [CHANGE] Cortex now rejects data with duplicate labels. Previously, such data was accepted, with duplicate labels removed with only one value left. #1964
 * [CHANGE] Changed the default value for `-distributor.ha-tracker.prefix` from `collectors/` to `ha-tracker/` in order to not clash with other keys (ie. ring) stored in the same key-value store. #1940
+* [FEATURE] Write-Ahead-Log added in ingesters for more data reliability against ingester crashes. #1103
+  * `--ingester.wal-enabled`: Setting this to `true` enables writing to WAL during ingestion.
+  * `--ingester.wal-dir`: Directory where the WAL data should be stored and/or recovered from.
+  * `--ingester.checkpoint-enabled`: Set this to `true` to enable checkpointing of in-memory chunks to disk.
+  * `--ingester.checkpoint-duration`: This is the interval at which checkpoints should be created.
+  * `--ingester.recover-from-wal`: Set this to `true` to recover data from an existing WAL.
+  * For more information, please checkout the ["Ingesters with WAL" guide](https://cortexmetrics.io/docs/guides/ingesters-with-wal/).
 * [FEATURE] The distributor can now drop labels from samples (similar to the removal of the replica label for HA ingestion) per user via the `distributor.drop-label` flag. #1726
 * [FEATURE] Added flag `debug.mutex-profile-fraction` to enable mutex profiling #1969
 * [FEATURE] Added `global` ingestion rate limiter strategy. Deprecated `-distributor.limiter-reload-period` flag. #1766
 * [FEATURE] Added support for Microsoft Azure blob storage to be used for storing chunk data. #1913
 * [FEATURE] Added readiness probe endpoint`/ready` to queriers. #1934
-* [FEATURE] EXPERIMENTAL: Added `/series` API endpoint support with TSDB blocks storage. #1830
 * [FEATURE] Added "multi" KV store that can interact with two other KV stores, primary one for all reads and writes, and secondary one, which only receives writes. Primary/secondary store can be modified in runtime via runtime-config mechanism (previously "overrides"). #1749
-* [FEATURE] EXPERIMENTAL: Added TSDB blocks `compactor` component, which iterates over users blocks stored in the bucket and compact them according to the configured block ranges. #1942
+* [FEATURE] Added support to store ring tokens to a file and read it back on startup, instead of generating/fetching the tokens to/from the ring. This feature can be enabled with the flag `-ingester.tokens-file-path` for the ingesters and `-ruler.tokens-file-path` for the ruler. #1750
```

Comment:
is this `ruler.ring` ?

Reply:
Actually I've just realized this was masked in the PR https://github.com/cortexproject/cortex/pull/1987, so it's not configurable (and I think it makes sense due to how the ruler uses the ring). I remove the mention from the changelog.